# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/ModuleLoader.cpp
# Commit: a71a2a98b6bc
# Full Hash: a71a2a98b6bc017a7bb60d4b7c41f543f156eede
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-21 21:39:46
# Regressor Bug: 1742437
# File Overlap Count: 1
# Description:
#   Bug 1742437 - move mElement from ScriptFetchOptions to DOMScriptLoadContext; r=jonco,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D136742
# ==============================================================================

diff -r d52858ae58d2 -r a71a2a98b6bc dom/script/ModuleLoader.cpp
--- a/dom/script/ModuleLoader.cpp	Mon Feb 21 15:39:04 2022 +0000
+++ b/dom/script/ModuleLoader.cpp	Mon Feb 21 15:39:05 2022 +0000
@@ -267,7 +267,8 @@
     }
 
     RefPtr<ModuleScript> moduleScript =
-        new ModuleScript(aRequest->mFetchOptions, aRequest->mBaseURL);
+        new ModuleScript(aRequest->mFetchOptions, aRequest->mBaseURL,
+                         aRequest->mLoadContext->mElement);
     aRequest->mModuleScript = moduleScript;
 
     if (!module) {